api/client.go

```diff
@@ -71,7 +74,12 @@ func (c *Client) Do(req *http.Request, v interface{}) (*http.Response, error) {
 	default:
 		if v != nil {
 			defer res.Body.Close()
-			if err := json.NewDecoder(res.Body).Decode(v); err != nil {
+
+			var bodyCopy bytes.Buffer
+			body := io.TeeReader(res.Body, &bodyCopy)
+
+			if err := json.NewDecoder(body).Decode(v); err != nil {
+				debug.Println("res.Body:", bodyCopy.String())
 				return nil, fmt.Errorf("error parsing API response - %s", err)
```

Comment:
How was the verbose logging activated in the above command? It doesn't look like a flag was passed to `./cli`.

I think you're right that we should always show the body in verbose mode, that would be immensely helpful. Maybe with a newline after it so that the "usual" output is separated a bit.